### PR TITLE
Fix PE-D Bugs

### LINKED ISSUE #297: [PE-D][Tester C] Inconsistent capitalisation
> The Architecture Diagram given above explains the high-level design of the App.

App is capitalised here but not in here:

> Main (consisting of classes Main and MainApp ) is in charge of the app launch and shut down

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: anselmlong/ped#32

### LINKED ISSUE #302: [PE-D][Tester C] Placeholders not removed
> [Proposed] Data archiving
{Explain here how the data archiving feature will be implemented}

This placeholder is not removed and may distract developers. 

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: anselmlong/ped#34

### LINKED ISSUE #303: [PE-D][Tester B] UG Command summary does not match some formats
Issue: some commands have formats that do not match the format in the features list
Suggestions: Change to ensure all command formats are correct

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Sornsornah/ped#14

### LINKED ISSUE #308: [PE-D][Tester C] User stories - Mismatched benefit
Why will your students drop out? This made me laugh though. I think a better phrasing could be when the student is no longer in your class.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: anselmlong/ped#37

### LINKED ISSUE #340: [PE-D][Tester D] StyleSheet missing errors while starting on CLI/linux
launching from a command terminal gives these additional warnings when starting up, doesn't affect the application functionality as far as I am aware but would be good to address as it may confuse users or result in the GUI being different from what you guys intended it look like

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: flyingsalsa/ped#21

### PR DESCRIPTION
**Pull request by jason-chuah**
Closes #340
Closes #308
Closes #303
Closes #302
Closes #297

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Examples:
 
 Adds an emergency contact's name to the specified person in the student list.
 
-Format: `addEcName INDEX en/[ECNAME]`
+Format: `addEcName INDEX [en/ECNAME]`
```
[the remark this answers is not in the document]
Comment (BunnyHoppp) [reply docs/UserGuide.md:185]: en/ is not optional but ECNAME is optional

File: docs/UserGuide.md [hunk 2]
```diff
@@ -410,13 +410,13 @@ _Details coming soon ..._
...
-| **Add Emergency Contact Name**   | `addEcName INDEX [en/EMERGENCY_CONTACT_NAME]` <br> e.g., `addEcName 1 en/John Doe`                                                                                                                                 |
-| **Add Emergency Contact Number** | `addEcNumber INDEX [ep/EMERGENCY_CONTACT_NUMBER]`<br> e.g., `addEcNumber 2 ep/91231234`                                                                                                                            |
+| **Add Emergency Contact Name**   | `addEcName INDEX [en/ECNAME]` <br> e.g., `addEcName 1 en/John Doe`                                                                                                                                                 |
+| **Add Emergency Contact Number** | `addEcNumber INDEX [ep/ECNUMBER]`<br> e.g., `addEcNumber 2 ep/91231234`                                                                                                                                            |
```
Comment (gohqingkhang) [inline docs/UserGuide.md:414]:
```suggestion
| **Add Emergency Contact Name**   | `addEcName INDEX en/ECNAME` <br> e.g., `addEcName 1 en/John Doe`                                                                                                                                                 |
| **Add Emergency Contact Number** | `addEcNumber INDEX ep/ECNUMBER`<br> e.g., `addEcNumber 2 ep/91231234`                                                                                                                                            |
```
should not be optional?

**Review by gohqingkhang — COMMENTED** [review]
small issue

File: docs/UserGuide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gohqingkhang) [reply docs/UserGuide.md:185]:
```suggestion
Format: `addEcName INDEX en/[ECNAME]`
```
corrected

**Pull request merged**